kairon/api/app/routers/bot/data.py

```
@@ -379,11 +380,34 @@ async def knowledge_vault_sync(
         message="Processing completed successfully",
         data=None
-    )
\ No newline at end of file
+    )
+
+
+@router.post("/integrations/add", response_model=Response)
+async def add_pos_integration_config(
+    request_data: POSIntegrationRequest,
+    sync_type: str,
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS),
```

Ruff (0.8.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)